### PR TITLE
Automating LogDNA Agent Build and Push for Windows

### PR DESCRIPTION
**Pull request by smusali**
### This PR contains:
* CircleCI Config Indentation Correction;
* New `build_windows` workflow containing `build_windows` and `push_to_choco` jobs which are to get triggered in each `/windows-[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?/` tags;
* Moving all Windows-specific files to `scripts/windows` folder;
* Renaming `license.txt` to `LICENSE.md`;
* Updated `nodejs` version to `10.15.3` in `.travis.yml` and `appveyor.yml`;
* Updating `ws` dependency version to `7.0.0` in order to solve `1006` errors.

### Requirements:
In order to have this code change work, the following environment variables have to be set:
* `AWS_ACCESS_KEY_ID`;
* `AWS_SECRET_ACCESS_KEY`;
* `CHOCO_API_KEY`.

### Notes:
I have built, packed, and pushed `v1.5.6` to `Chocolatey` using this configuration.

### CODE REVIEW

**Comment by smusali** [comment]
@dchai76, it's mainly because of the dependencies - I remember, once I was testing multiple versions and the only version `bufferutil` was getting installed successfully was `8.11.3` - upgrading `bufferutil` was also breaking some functionalities. We can try by changing `8.11.3` to `10.0.0` but I'm afraid it'll break the agent. Also, I think it should be focus of other `Pull Request` - in other words, we can deal with this separately.

**Comment by dchai76** [comment]
I only ask because you're updating the Node version used for build anyway, and because of that, in my mind we might as well move to a more current version, especially since we're in the process of moving to Node 10 everywhere else in our system. But I'll leave this as your call.

**Comment by RYadav95** [comment]
Looks good to me.

**Comment by smusali** [comment]
@dchai76, the requested change is done!

@gregswift @RYadav95 @leeliu @baronomasia, can you guys review it again and approve?

Thanks in advance!

File: scripts/windows/VERIFICATION.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+ 
+To verify the logdna-agent.exe:
```
Comment (dchai76) [inline scripts/windows/VERIFICATION.txt]: Can you remove the `the` here? Just "To verify logdna-agent.exe:"

File: scripts/windows/VERIFICATION.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ 
+1. Download <https://s3.amazonaws.com/repo.logdna.com/win/logdna-agent.latest.zip> which includes "logdna-agent.exe"
+2. You can use one of the following methods to obtain the checksum
+  - Use powershell function 'Get-Filehash'
+  - Use chocolatey utility 'checksum.exe'
+ 
+  checksum type: sha256
+  checksum: 9daf22c15f4f42e8c7ac4cdf73c26f7154d325d0defe14ece18d1203a9c7afd1
+ 
+This package is published by the LogDNA. You can build a binary directly
```
Comment (dchai76) [inline scripts/windows/VERIFICATION.txt]: Remove "the" here also, should be "This package is published by LogDNA."

File: scripts/windows/VERIFICATION.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+by following instructions here:
```
Comment (dchai76) [inline scripts/windows/VERIFICATION.txt]: Here you need to *insert* a "the", so "by following the instructions here:"

Comment (smusali) [reply scripts/windows/VERIFICATION.txt]: Done!

File: scripts/windows/VERIFICATION.txt (hunk 2 above)
Comment (smusali) [reply scripts/windows/VERIFICATION.txt]: Done!

File: scripts/windows/VERIFICATION.txt (hunk 1 above)
Comment (smusali) [reply scripts/windows/VERIFICATION.txt]: Done!

File: scripts/windows/VERIFICATION.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+ 
+To verify logdna-agent.exe:
+ 
+1. Download <https://s3.amazonaws.com/repo.logdna.com/win/logdna-agent.latest.zip> which includes "logdna-agent.exe"
+2. You can use one of the following methods to obtain the checksum
```
Comment (dchai76) [inline scripts/windows/VERIFICATION.txt]: Sorry one more, I think "calculate the checksum" is slightly better than "obtain the checksum".

Comment (smusali) [reply scripts/windows/VERIFICATION.txt]: You are right! Done!

File: .travis.yml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '6.10.2'
+- '8.11.3'
```
Comment (dchai76) [inline .travis.yml]: Sorry, this is different from the `nodejs_version` in `appveyor.yml` below; intentional?

Comment (smusali) [reply .travis.yml]: Wait, hasn't it got updated? How come? It shows updated on my local.

Comment (smusali) [reply .travis.yml]: Done!

File: .travis.yml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '6.10.2'
+- '10.15.3'
```
Comment (dchai76) [inline .travis.yml]: Sorry, I know I'm being an extreme nitpicker here, but this is the only place in any `.yml` file where we use single quotes, everywhere else is double. Can we be consistent?

Comment (smusali) [reply .travis.yml]: What if I remove all the quotes? It won't matter and it's again for keeping consistency

Comment (smusali) [reply .travis.yml]: Done!

**Pull request merged**